src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -154,7 +154,7 @@ public void search() {
         if (CompositeIdFetcher.containsValidId(query)) {
             CompositeIdFetcher compositeIdFetcher = new CompositeIdFetcher(preferencesService.getImportFormatPreferences());
             parserResultCallable = () -> new ParserResult(OptionalUtil.toList(compositeIdFetcher.performSearchById(query)));
-            fetcherName = compositeIdFetcher.getName();
+            fetcherName = "Identifier-based Web Search";
```

Comment:
Would be great to have this localized as well

Reply:
Added to l10n

Reply:
Self-directed learning comment: https://devdocs.jabref.org/code-howtos/localization.html

Direct comment: You forgot to put `Localization.lang(...)` around. JabRef has no magic to determine translation strings.